docs/backend-requests/handling/manual-jwt.mdx

```diff
@@ -7,37 +7,20 @@ description: Learn how to manually verify Clerk-generated session tokens (JWTs).
 
-For every request, you must validate its token to make sure it hasn't expired and it is authentic (i.e. no malicious user tried to tamper with it). If these validations pass, then it means that the user is authenticated to your application and you should consider them signed in.
-
 <Steps>
 
-### Retrieve the session token
+### Initialiaze a Clerk client
 
-Retrieve the session token from either `__session` cookie for a same origin request or from the Authorization header for cross origin requests.
+Initialize a Clerk client using the Clerk Secret Key and the Clerk Publishable Key provided in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.
 
-### Get your instance's public key
+### Verify request
 
-There are three ways to obtain your public key:
+To verify the request you should check that the `isSignedIn` attribute of the `clerkClient.authenticateRequest(request)` return value is `true`. The `request` parameter provided can be any object extending `Request`.
 
-1. Using the Backend API in JSON Web Key Set (JWKS) format at the following endpoint [https://api.clerk.com/v1/jwks](https://clerk.com/docs/reference/backend-api/tag/JWKS#operation/GetJWKS).
-
-2. Using the Frontend API in JSON Web Key Set (JWKS) format at the following endpoint `https://<YOUR_FRONTEND_API>/.well-known/jwks.json`. This can be obtained from the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. Scroll down and click on **Advanced** and in the **JWT public key** section, copy the **JWKS URL**.
+The `clerkClient.authenticateRequest(request)` executes the following steps:
 
-<Images
-  src="/docs/images/backend-requests/jwt-public-key.jpg"
-  width={3024}
-  height={1650}
-  alt="The 'API Keys' page in the Clerk Dashboard. There is a red box around the 'JWT public key' section, with a red arrow pointing to the 'JWKS URL' tab of the code example."
-/>
-
-3. Using the PEM public key provided in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. Scroll down and click on **Advanced** and in the **JWT public key** section, copy the **PEM public key**. This option should only be used as a fallback for when the first two options are not available.
-
-### Verify the token signature
-
-To verify the token signature, you should:
-
-1. Use the above Public Key to verify the token's signature.
-2. Validate that the token is not expired by checking the `exp` ([Expiration time](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4)) and `nbf` ([Not before](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5)) claims.
+1. Uses the instance Public Key to verify the token's signature (to manually find and use the public key check [here](#get-your-instances-public-key)).
```

Comment:
so does authenticateRequest automatically have access to the user's public key?

Reply:
The `authenticateRequest()` retrieves the jwks (containing the JWK to verify to the token signature) of the instance from BAPI using the `secretKey` parameter.